docker/gcp_pudl_etl.sh

```diff
@@ -12,128 +12,141 @@ function send_slack_msg() {
+    fi
 }
 
 function zenodo_data_release() {
     echo "Creating a new PUDL data release on Zenodo."
-    ~/devtools/zenodo/zenodo_data_release.py --publish --env sandbox --source-dir $PUDL_OUTPUT
+    ~/devtools/zenodo/zenodo_data_release.py --publish --env sandbox --source-dir "$PUDL_OUTPUT"
 }
 
-
 function notify_slack() {
     # Notify pudl-builds slack channel of deployment status
-    if [ $1 = "success" ]; then
+    if [ "$1" = "success" ]; then
         message=":large_green_circle: :sunglasses: :unicorn_face: :rainbow: The deployment succeeded!! :partygritty: :database_parrot: :blob-dance: :large_green_circle:\n\n "
         message+="<https://github.com/catalyst-cooperative/pudl/compare/main...${GITHUB_REF}|Make a PR for \`${GITHUB_REF}\` into \`main\`!>\n\n"
-    elif [ $1 = "failure" ]; then
+    elif [ "$1" = "failure" ]; then
         message=":large_red_square: Oh bummer the deployment failed ::fiiiiine: :sob: :cry_spin:\n\n "
     else
         echo "Invalid deployment status"
         exit 1
     fi
-    message+="See https://console.cloud.google.com/storage/browser/nightly-build-outputs.catalyst.coop/$ACTION_SHA-$GITHUB_REF for logs and outputs."
+    message+="See https://console.cloud.google.com/storage/browser/nightly-build-outputs.catalyst.coop/$BUILD_ID for logs and outputs."
 
     send_slack_msg "$message"
 }
 
 # # Run ETL. Copy outputs to GCS and shutdown VM if ETL succeeds or fails
 # 2>&1 redirects stderr to stdout.
-run_pudl_etl 2>&1 | tee $LOGFILE
+run_pudl_etl 2>&1 | tee "$LOGFILE"
 
 ETL_SUCCESS=${PIPESTATUS[0]}
 
 copy_outputs_to_gcs
 
 # if pipeline is successful, distribute + publish datasette
 if [[ $ETL_SUCCESS == 0 ]]; then
+    if [ "$GITHUB_ACTION_TRIGGER" = "schedule" ]; then
+        # Remove read-only authentication header added by git checkout
+        git config --unset http.https://github.com/.extraheader
+        git config user.email "pudl@catalyst.coop"
+        git config user.name "pudlbot"
+        git remote set-url origin "https://pudlbot:$PUDL_BOT_PAT@github.com/catalyst-cooperative/pudl.git"
+        # Update the nightly branch to point at newly successful nightly build tag
+        git checkout nightly
+        git merge --ff-only "$NIGHTLY_TAG"
+        git push
+    fi
     # Deploy the updated data to datasette
-    if [ $GITHUB_REF = "dev" ]; then
-        python ~/devtools/datasette/publish.py 2>&1 | tee -a $LOGFILE
+    if [ "$GITHUB_REF" = "dev" ]; then
+        python ~/devtools/datasette/publish.py 2>&1 | tee -a "$LOGFILE"
         ETL_SUCCESS=${PIPESTATUS[0]}
     fi
 
     # Compress the SQLite DBs for easier distribution
     # Remove redundant multi-file EPA CEMS outputs prior to distribution
-    gzip --verbose $PUDL_OUTPUT/*.sqlite && \
-    rm -rf $PUDL_OUTPUT/hourly_emissions_epacems/ && \
-    rm -f $PUDL_OUTPUT/metadata.yml
+    gzip --verbose "$PUDL_OUTPUT"/*.sqlite && \
+    rm -rf "$PUDL_OUTPUT/core_epacems__hourly_emissions/" && \
```

Comment:
Whoops, manual renaming of things succccks.  Gotta go delete these directories in the deployed outputs.